### ISSUE TITLE
Newline is not appended to error messages

### ISSUE THREAD

**Issue by MorrisLaw** [jfeliu007/goplantuml#93]
Due to how `fmt.Errorf()` formats strings, unlike `fmt.Println()`, a newline is not automatically appended to the string. This leads to things like:

e.g. `%s is not a directory`:
```
jeremys-macbook:~ jeremymorris$ goplantuml ./some_file.py
usage:
goplantum <DIR>
DIR Must be a valid directory
./some_file.py is not a directoryjeremys-macbook:~ jeremymorris$
```

e.g. `could not find directory %s`:
```
jeremys-macbook:~ jeremymorris$ goplantuml help
usage:
goplantum <DIR>
DIR Must be a valid directory
could not find directory helpjeremys-macbook:~ jeremymorris$
```

*NOTE:* Notice the `goplantum <DIR>` in my examples. That was also in the output, it looks to be a typo.

**Comment by jfeliu007** [comment]
Thanks for opening this issue.

**Comment by jfeliu007** [comment]
Thanks again. I will mark this as resolved by #96
